docs/changelog/89212.yaml

```diff
@@ -3,3 +3,27 @@ summary: Add an unattended mode setting to transform
 area: Transform
 type: enhancement
 issues: []
+highlight:
+  title: Unattended mode for transforms
+  body: |-
+    The new `unattended` setting for transforms introduces unattended mode. It 
+    enables the transform to retry indefinitely, even in the face of errors that 
+    require direct intervention to resolve. For example, a high disk watermark 
+    may prevent all ingest activities, including the indexing operation of a 
+    transform; however once the original issue is resolved by the system 
+    administrator, the transform automatically continue to operate from where it 
+    stopped working. Neither the system administrator nor the end-user need to 
+    restart the transform.
+
+    [discrete]
+    [[frequent-items-agg]]
+    === Frequent items aggregation
```

Comment:
Is the way this has been done going to mean the frequent items aggregation shows up as a transform feature?

Reply:
@droberts195 
It won't show up as a transform feature. 
The items in the PR will show up as two distinct sections on the What's new page. For example, in the [8.4 highlights](https://www.elastic.co/guide/en/elasticsearch/reference/current/release-highlights.html#composite_aggregations_in_datafeeds_are_generally_available), the sections about composite aggs and the optimized DFA speed was added in the same PR (similar to this situation). The only indicator that the two items were added in the same PR is the pull request reference after the second section. The workflow is far from ideal, we have an [issue](https://github.com/elastic/platform-docs-team/issues/38) for improving the process, so hopefully, we don't need to publish the release highlights in this "hacky" way in the future.